docs/release-notes/release-notes-0.15.2.md

```diff
@@ -16,6 +16,8 @@
 
 ## Taproot
 
+* [Add `p2tr` address type to account
+  import](https://github.com/lightningnetwork/lnd/pull/6965).
```

Comment:
6966?

Reply:
Oops, my guessing game wasn't accurate. Thanks for catching it, fixed.